Readme.md

```diff
@@ -63,10 +63,30 @@ Each eXistdb installation to monitor is added as an instance entry at */db/apps/
 In the Monex Remote Monitoring tab click "Run" to start all remote monitoring jobs. You should now see an entry "localhost" beneath "Remote Monitoring" and beneath that an entry "Timelines".
 
 
+Building
+--------
+If you wish to build Monex from source code you should follow these steps:
 
- 
+1. Clone it from *wolfgangmm* (or from your own fork) `git clone https://github.com/wolfgangmm/monex.git`.
```

Comment:
Sorry, should now be `https://github.com/eXide/monex.git`

Reply:
Ah right. I have fixed that now.